<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="StartSimulation">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>460</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Simulation</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;Elevator System&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fireSafety">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Fire</string>
    </property>
   </widget>
   <widget class="QPushButton" name="powerOutage">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Power Outage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HelpButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>280</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doorObstacle">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Door Obstacle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="overload">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Overloaded</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="OutputBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>231</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string># of Floor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string># of Car</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Weight</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Weight</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Elevator Panel</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="DisplayWindow">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>190</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>170</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>PASSENGER LIST</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Up">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>UP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Down">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>DOWN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>System Log</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor Panel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>20</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numberFloor">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>80</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="numberCar">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>394</y>
      <width>72</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="DisplayWindow_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="CurrentWeight">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0 Kg</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numberFloor_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>200 Kg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MainUseCase">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>70</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>1st Allocation Strategy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Open">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>DoorOpen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Close">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>DoorClose</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MainUseCase_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>100</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>2nd Allocation Strategy</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>390</y>
      <width>72</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>400</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Destn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Down_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Disable Destn</string>
    </property>
   </widget>
   <zorder>DisplayWindow_2</zorder>
   <zorder>textBrowser_6</zorder>
   <zorder>StartSimulation</zorder>
   <zorder>textBrowser_2</zorder>
   <zorder>fireSafety</zorder>
   <zorder>powerOutage</zorder>
   <zorder>HelpButton</zorder>
   <zorder>doorObstacle</zorder>
   <zorder>overload</zorder>
   <zorder>OutputBox</zorder>
   <zorder>textBrowser_3</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>DisplayWindow</zorder>
   <zorder>label_7</zorder>
   <zorder>Up</zorder>
   <zorder>Down</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>label_11</zorder>
   <zorder>pushButton_10</zorder>
   <zorder>numberFloor</zorder>
   <zorder>numberCar</zorder>
   <zorder>comboBox_3</zorder>
   <zorder>CurrentWeight</zorder>
   <zorder>numberFloor_3</zorder>
   <zorder>MainUseCase</zorder>
   <zorder>Open</zorder>
   <zorder>Close</zorder>
   <zorder>MainUseCase_2</zorder>
   <zorder>comboBox_4</zorder>
   <zorder>label_10</zorder>
   <zorder>Down_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_10</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>720</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>742</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
